authors/apps/authentication/tests/test_base.py

```diff
@@ -1,13 +1,17 @@
 """
 Test base module
 """
-from rest_framework.test import APITestCase, APIClient
+from django.test import TestCase
+from rest_framework.test import APIClient
+
+from authors.apps.authentication.models import User
 from authors.apps.authentication.tests.test_data import (
-    login_credentials_data, 
+    login_credentials_data,
     login_data
-    )
+)
+
 
-class BaseTestCase(APITestCase):
+class BaseTestCase(TestCase):
```

Comment:
@felixkiryowa Please undo the changes in this `test_base.py` file.

Reply:
I  have fixed  it

Reply:
@felixkiryowa these changes still exist in your codebase. Please fix.